packages/expo-notifications/android/src/main/java/expo/modules/notifications/service/NotificationsService.kt

```diff
@@ -497,7 +498,18 @@ open class NotificationsService : BroadcastReceiver() {
       return intent
     }
 
-    fun getNotificationResponseFromIntent(intent: Intent): NotificationResponse? {
+    fun getNotificationResponseFromBroadcastIntent(intent: Intent): NotificationResponse {
+      val notification = intent.getParcelableExtra<Notification>(NOTIFICATION_KEY)!!
+      val action = intent.getParcelableExtra<NotificationAction>(NOTIFICATION_ACTION_KEY)!!
```

Comment:
Could you refactor that part not to use `!!` here? We can't be fully sure it's non-null 😅